QuantLib/ql/time/calendars/israel.cpp

```diff
@@ -322,7 +322,7 @@ namespace QuantLib {
             || ((d == 4  || d == 5 ) && m == October   && y == 2015)
             || ((d == 23 || d == 24) && m == October   && y == 2016)
             || ((d == 11 || d == 12) && m == October   && y == 2017)
-            || ((d==30&&m==September)||(d==1&&m==October)&&y== 2018)
+            || ((d==30&&m==September)||((d==1&&m==October)&&y== 2018))
```

Comment:
This should be grouped differently; it's (30 sep || 1 oct) && 2018. The same goes for the two that follow.